### PR TITLE
Add README for amp-experiment.

### PR DESCRIPTION
**Pull request by lannka**
#1411

### CODE REVIEW

**Comment by lannka** [comment]
@rudygalfi You probably want to pay more attention to those fields added after design review: `group`, `sticky`.

File: extensions/amp-experiment/amp-experiment.md [hunk 1]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
 
 # <a name="amp-experiment"></a> `amp-experiment`
 
-Under development. See [issue #1411](https://github.com/ampproject/amphtml/issues/1411).
+| | |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: shows as empty row -- eliminate?

File: extensions/amp-experiment/amp-experiment.md [hunk 2]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
...
+|-----------------------|------------------------------|
+| **Description** | Conduct user experience experiments on an AMP document and collect corresponding data with `amp-pixel` or `amp-analytics` | 
+| **Availability** | In Experiment |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: "Experimental" - also link that word to https://cdn.ampproject.org/experiments.html

File: extensions/amp-experiment/amp-experiment.md [hunk 3]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
...
+    }
+  </script>
+</amp-experiment>
+```
+
+At top level, the JSON is a map of experiment configurations keyed by experiment names. In each experiment, available settings are described in the table below:
+
+| Name                  | Is required field?           | Description |
+|-----------------------|------------------------------|-------------|
+|`sticky`               | No, default=`true`           | Whether the experiment is user sticky or not. |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: Whether the experiment assignment is sticky for a user or not.

File: extensions/amp-experiment/amp-experiment.md (hunk 1 above)
Comment (lannka) [inline extensions/amp-experiment/amp-experiment.md]: That's an [unfortunate](http://stackoverflow.com/questions/17536216/create-table-without-header-in-markdown)... 

If this really annoys you, I can switch to HTML `<table>` ..

File: extensions/amp-experiment/amp-experiment.md [hunk 4]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
...
+|`consentNotificationId`| No, default=`undefined`      | The element ID of the `amp-user-notification` to be dismissed before a user sticky can be conducted. To not block the page rendering, an experiment with this field specified will be skipped if the consent is not provided prior to the current visit. That’s to say, only returning visits with user consent can trigger such an experiment. It only works when `sticky=true`. |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: "before a sticky experiment can be"

Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: At the end, would be more appropriate to say: "This setting is only relevant when when `sticky=true`."?

File: extensions/amp-experiment/amp-experiment.md [hunk 5]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
...
+|`variants`             | Yes                          | A name-to-percentage map where percentage is a float number in range (0, 100) that indicates the amount of traffic will be allocated to the variant. Variants don’t have to sum up to 100%. In that case, there’ll be a portion of the traffic allocated to none, which is a reserved keyword that indicates “no variant allocated”. |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: allocated to a variant named "none", which is a reserved keyword that indicates no variant was allocated.

File: extensions/amp-experiment/amp-experiment.md [hunk 6]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
...
+| **Advanced settings** |||
+|`cidScope`             | No, default=`amp-experiment` | The CID scope for user sticky experiment. Only useful when you want to reuse an existing CID. It only works when `sticky=true`. |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: See above comment re: phrasing as "only relevant when..."

Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: link CID scope to: https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#client_id

File: extensions/amp-experiment/amp-experiment.md [hunk 7]
```diff
@@ -16,4 +16,90 @@ limitations under the License.
...
+|`group`                | No, default=`{experimentName}` | Experiments with the same group name will share the same CID space. Only useful when multiple experiments want to have correlated user grouping. It only works when `sticky=true`. |
```
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: Again, only relevant when

**Comment by rudygalfi** [comment]
Reviewed, left some comments. LGTM once those are addressed.

**Comment by lannka** [comment]
Done. PTAL

File: extensions/amp-experiment/amp-experiment.md (hunk 1 above)
Comment (rudygalfi) [inline extensions/amp-experiment/amp-experiment.md]: Ah, yes, you should use the format in other docs like https://github.com/ampproject/amphtml/edit/master/extensions/amp-analytics/amp-analytics.md.

These files get processed into the documentation files on ampproject.org/docs, and I believe the markdown table breaks it.

Comment (lannka) [inline extensions/amp-experiment/amp-experiment.md]: Fixed.

**Comment by rudygalfi** [comment]
LGTM

**Comment by lannka** [comment]
Thanks for reviewing.

**Pull request merged**